docs/source/standard/auction.rst

```diff
@@ -22,86 +21,119 @@ Schema
+
   The name of the auction, displayed in listings. 
-  Originates from `lot.title`.
  
 :description:
   string, multilingual, required
 
+  Originates from `lots.description`.
+  
   Detailed auction description. 
-  Originates from `lot.description`.
-    
+  
 :procurementMethodType:
   string, auto-generated, required
-    
-  Type of the procedure within the auction announcement. Possible values:
-
-   :sellout.english: 
-    - procedure with the open ascending price auction;
-   :sellout.insider: 
-    - procedure with the insider auction.
-   
-  Originates from `lot.auctions.procurementMethodType`.
-     
+  
+  Originates from `lots.auctions.procurementMethodType`.
+  
+  Type of the procedure within the auction announcement. The only type is english. 
+
+:procurementMethodDetails:
+  string, auto-generated
+  
+  Originates from `lots.auctions.procurementMethodDetails`.
+
+  Parameter that accelerates auction periods. Set quick, accelerator=1440 as text value for procurementMethodDetails for the time frames to be reduced in 1440 times.
+
+:submissionMethodDetails:
+  
+  string, auto-generated
+  
+  Originates from `lots.auctions.submissionMethodDetails`.
+
+  Parameter that works only with mode = “test” and speeds up auction start date.
+
+  Possible values are:
+
+  * -`quick(mode:no-auction)`;
+  * -`quick(mode:fast-forward)`.
+
+
 :procuringEntity:
   :ref:`ProcuringEntity`, required
 
+  Originates from `lots.lotCustodian`.
+  
   Organization conducting the auction.
-  Originates from `lot.lotCustodian`.
-   
+
   |ocdsDescription|
   The entity managing the procurement, which may be different from the buyer who is paying / using the items being procured.
 
 :tenderAttempts:
   integer, required
 
-  The number which represents what time (from 1 up to 3) procedure with a current lot takes place
-  Originates from `lot.auctions.tenderAttempts`
+  Originates from `lots.auctions.tenderAttempts`.
+
+  The number which represents what time (from 1 up to 3) procedure with a current lot takes place.
 
 :value:
   :ref:`value`, required
 
+  Originates from `lots.auctions.value`. 
+
+  
   Total available budget of the 1st auction. Bids lower than ``value`` will be rejected.
 
-  Auction.value for 2nd and 3rd auctions within the privatization cycle will be calculated as half of the auction.value provided.
+  Auction.value for 2nd auction within the privatization cycle will be calculated as half of the auction.value provided.
 
   |ocdsDescription|
   The total estimated value of the procurement.
   
-  Originates from `lots.auctions.value`. 
-
 :guarantee:
   :ref:`Guarantee`, required
 
-  Bid guarantee. Lots.auctions.guarantee for 2nd and 3rd auctions within the privatization cycle will be calculated automatically.
-  Originates from `lot.auctions.guarantee`.
+  Originates from `lots.auctions.guarantee`.
+
+  Bid guarantee. Lots.auctions.guarantee for 2nd auction within the privatization cycle will be calculated automatically.
```

Comment:
Lots.auctions.guarantee for 2nd auction within the privatization cycle will be calculated automatically. - this one should be removed